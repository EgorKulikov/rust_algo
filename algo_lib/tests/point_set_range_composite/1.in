5 5
1 2
3 4
5 6
7 8
9 10
1 0 5 11
1 2 4 12
0 1 13 14
1 0 4 15
1 2 5 16
