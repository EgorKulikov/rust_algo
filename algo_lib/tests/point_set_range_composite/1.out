14005
470
8275
5500
